susemanager/src/mgr_bootstrap_data.py

```diff
@@ -1512,6 +1512,26 @@
         "PKGLIST": PKGLIST15_SALT + PKGLIST15_X86_ARM,
         "DEST": DOCUMENT_ROOT + "/pub/repositories/opensuse/15/6/bootstrap/",
     },
+    "openSUSE-Leap-16.0-x86_64-uyuni": {
+        "BASECHANNEL": "opensuse_leap16_0-x86_64",
+        "PKGLIST": PKGLISTMICRO_BUNDLE_ONLY,
```

Comment:
I am tempted to create a group called `PKGLIST_BUNDLE_ONLY`... any new OS will only support bundle, regardless of being `Micro` or not... right?